scripts/require.sh

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
+# File used to check dependencies
+
+if [ ! -f composer.phar ]; then
+    echo "composer.phar not found, we'll see if composer is installed globally."
```

Comment:
and if `composer.phar` is found, we need to use it in the command, instead of `composer`.